book/website/collaboration/ppie-events/ppie-events-comms.md

```diff
@@ -0,0 +1,363 @@
+Instead, consider setting a date when you or the research team will send an update, regardless of whether you have completed follow-up materials or whether a proposed project receives funding. 
+
+(cl-ppie-events-comms-conclusion)=
+## Conclusion
+By demonstrating an open channel of communication about accessibility, and by being thorough in your communications plans, you will have a successful event that incorporates the expertise and experience of people with lived experience.
+
+(cl-ppie-events-comms-key-information)=
+## Key Information Checklist
+This is not an exhaustive list of what to include in core communication, however, it can be used as a reference in addition to the information above for what to include on a website or in an information pack. 
+
+* **Information about the event** including:
+    * The Date and Time of the event.
+    * Event Agenda or Program.
+    * Location of the event.
+      Consider also sharing a map of the location and accompanying text instructions of the location and consider including details such as where parking would be available, where each of the session locations would be, and where to go in an emergency situation. 
+    * Information about transport links and suggestions for how to get to the location from public transport.
+    * Who is hosting the event and a point of contact. Consider including information about the hosts or supporting organizations.
+
+* Information about any **Reimbursement or Honorarium Policies** you may have. This includes details on: 
+    * What can be reimbursed.
+    * How to complete the process, as well as links to any forms that are needed. Consider writing a guide or instructions for any forms you may have. 
+    * What is involved in the process.
+    * Who to contact about reimbursements.
+      When working with large research teams, it can become unclear who is responsible for what tasks.
+      Providing details on the reimbursement process, and who to contact if there are questions, can help make people with lived experience who take part in your event feel supported and understood and establish a clear expectation.
+      This is especially important if the member of your event team who is the main point of contact for accessibility questions is different from the person able to process financial requests. 
+
+* **Contact information for support** or regarding special requests such as:
+    * The Event organiser. This may be the designated point of contact in an emergency.
+    * A person who is able to provide Accessibility information and help if you are able to support it for your event.
+      Try to make this contact available as soon as you announce the event, even if it is setting up a designated inbox.
+    * Any Catering or dietary requirements.
+      Because dietary requirements are protected information, not all members of an organizing team will have access to it or be able to answer questions. 
```

Comment:
It might be worth adding in a previous section some information about "protected characteristics" as they are used in the UK & USA for example, but not not apply more broadly, or being known as a term.